src/components/CarouselDots.js

```diff
@@ -17,14 +17,14 @@ class CarouselDots extends Component {
     const numberOfSlides = this.props.number || this.props.thumbnails.length;
```

Comment:
you have a typo in the branch name:

`fix/corrected-sematic-of-html-tags-in-dots-component`

`sematic` -> `semantic`

Reply:
Is any chance to correct it during the review?

Reply:
I guess you need to open a new PR